### PR TITLE
FT: Add example of using Account ID in ARN

### PR DESCRIPTION
**Pull request by bennettbuchanan**
Depends on:
* https://github.com/scality/Arsenal/pull/260
* https://github.com/scality/Integration/pull/530

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,9 +109,26 @@ Server is running.
...
+
+    ```
+    "Resource": "arn:scality:utapi:::buckets/foo"
+    ```
+
+    Sample ARN limiting the user's access to metrics of an account with the ID
+    '012345678901':
+
+    ```
+    "Resource": "arn:scality:utapi::012345678901:buckets/*"
```
Comment (rahulreddy) [inline README.md]: Let's change the default policy to include accountId. And here we can refer to on how to access metrics for all accounts.

File: README.md [hunk 2]
```diff
@@ -102,16 +102,69 @@ Server is running.
...
-    The user can also be restricted to only some buckets
-    (foo bucket for example) by changing the `Resource` property to
-    `"Resource": "arn:scality:utapi:::buckets/foo"`
+    In the above sample, the `Resource` property includes a series of Amazon
+    Resource Names (ARNs) used to define which resources the policy applies to.
+    Thus the sample policy applies to a user with an account ID '012345678901',
+    and grants access to metrics at the levels 'buckets', 'accounts', and
+    'users'.
+
+    The account ID of the ARN at any metric level can also be omitted. Extending
```
Comment (electrachong) [inline README.md]: If you omit it, does that mean any account can access those resources?

Comment (bennettbuchanan) [inline README.md]: Yes. Just updated to clarify.

File: README.md [hunk 3]
```diff
@@ -102,16 +102,69 @@ Server is running.
...
-    The user can also be restricted to only some buckets
-    (foo bucket for example) by changing the `Resource` property to
-    `"Resource": "arn:scality:utapi:::buckets/foo"`
+    In the above sample, the `Resource` property includes a series of Amazon
+    Resource Names (ARNs) used to define which resources the policy applies to.
+    Thus the sample policy applies to a user with an account ID '012345678901',
+    and grants access to metrics at the levels 'buckets', 'accounts', and
+    'users'.
+
+    The account ID of the ARN at any metric level can also be omitted, meaning
```
Comment (electrachong) [inline README.md]: The account ID of the ARN ... can also be omitted, allowing any account to access metrics for those resources. As an example, we can extend the above sample policy to allow any account to access metrics at the level 'service':

Comment (bennettbuchanan) [inline README.md]: Sounds good 👍 . Updated.

**Pull request merged**